src/components/implementation/no_interface/vkernel/vk_api.h

```diff
@@ -4,10 +4,10 @@
 #include "vk_types.h"
 
 /* extern functions */
-extern void vm_exit(void *);
 extern void vm_init(void *);
 extern void dom0_io_fn(void *);
 extern void vm_io_fn(void *);
+extern void *__inv_vkernel_serverfn(int a, int b, int c);
```

Comment:
OK, so how do you want to go forward with this?  Will other people be able to use any of this code?  Will you be able to use this code for your research (this paper)?  It is obvious we need to make this general at some point (i.e. the next level of libraries above cos_kernel_api + cos_defkernel_api + sl).  When do we do that?